projenrc/benchmark-test.ts

```diff
@@ -146,10 +143,13 @@ export class BenchmarkTest {
             },
           },
           {
-            name: 'Get TSC version',
-            id: 'get_tsc_version',
+            name: 'Get version line',
+            id: 'version',
             if: `github.event.repository.fork == false && github.ref == 'refs/heads/main'`,
-            run: `echo "TSC_VERSION=$(tsc --version | awk '{print $2}')" >> $GITHUB_ENV`,
+            run: [
+              `VERSION=$(node -p "require('./lib/version.js').RELEASE_LINE")`,
+              'echo "release-line=${RESULT}" >> $GITHUB_OUTPUT',
```

Comment:
To sense check, you might want to echo the version line here as well so it can be inspected.
```suggestion
              'echo $VERSION',
              'echo "release-line=${RESULT}" >> $GITHUB_OUTPUT',
```